src/index.ts

```diff
@@ -23,12 +27,17 @@ export const CustomColors: {
   configVar: chalk.green,
   release: chalk.blue.bold,
   cmd: chalk.cyan.bold,
+  pipeline: chalk.green.bold,
   app: (s: string) => color.heroku(`⬢ ${s}`),
   heroku: (s: string) => {
     if (!color.supports) return s
     let has256 = color.supports.has256 || (process.env.TERM || '').indexOf('256') !== -1
     return has256 ? '\u001b[38;5;104m' + s + ansiStyles.reset.open : chalk.magenta(s)
   },
+  stripColor: (s: string) =>
+    deprecate(() => {
+      return stripAnsi(s)
```

Comment:
can we just require it here so we don't have to load it?

Reply:
10-4